event/consumer_test.go

```diff
@@ -18,25 +17,30 @@ import (
 	. "github.com/smartystreets/goconvey/convey"
 )
 
+var ctx = context.Background()
+
 func TestConsume_UnmarshallError(t *testing.T) {
-	Convey("Given an event consumer with an invalid schema and a valid schema", t, func() {
+	Convey("Given an event consumer with an invalid schema and a valid schema", t, func(c C) {
 
 		reporter := reportertest.NewImportErrorReporterMock(nil)
-		messages := make(chan kafka.Message, 2)
-		messageConsumer := kafkatest.NewMessageConsumer(messages)
+		// messages := make(chan kafka.Message, 2)
```

Comment:
Remove commented code if no longer needed

Reply:
thanks, I forgot to do it. I will remove all the commented code I find in this file.